Dockerfile

```diff
@@ -1,10 +1,10 @@
 FROM golang:1.3
 
-ADD . /go/src/github.com/docker/swarm
+COPY . /go/src/github.com/docker/swarm
 WORKDIR /go/src/github.com/docker/swarm
 
-RUN go get github.com/tools/godep
-RUN godep go install
+ENV GOPATH $GOPATH:/go/src/github.com/docker/swarm/Godeps/_workspace
+RUN go install -v
 
 EXPOSE 2375
-ENTRYPOINT ["swarm"]
+CMD ["swarm"]
```

Comment:
@jfrazelle "because we should use the best practices for our Dockerfiles"

https://docs.docker.com/articles/dockerfile_best-practices/#entrypoint :wink: